pkg/aws/appmesh.go

```diff
@@ -224,25 +224,43 @@ func (v *VirtualNode) Listeners() []appmeshv1beta1.Listener {
 	}
 
 	var listeners = []appmeshv1beta1.Listener{}
-	for _, appmeshListener := range v.Data.Spec.Listeners {
+	for _, sdkListener := range v.Data.Spec.Listeners {
```

Comment:
I like the clarification of "sdk" here vs appmesh.

Reply:
I'm glad! It took some restraint to not apply it in other code, but I'll look to sticking to that for what I touch.